python/ray/tests/test_reference_counting.py

```diff
@@ -361,10 +363,13 @@ def get_large_object(self):
 # Remote function takes serialized reference and doesn't hold onto it after
 # finishing. Referenced object shouldn't be evicted while the task is pending
 # and should be evicted after it returns.
-def test_basic_serialized_reference(one_worker_100MiB):
-    @ray.remote
+@pytest.mark.parametrize("failure", [False, True])
+def test_basic_serialized_reference(one_worker_100MiB, failure):
+    @ray.remote(max_retries=0)
```

Comment:
Actually it would be great to also have some tests where `max_retries > 0` so we can make sure that ref counting works when there are retries.

Reply:
Oh yeah that's a really good point - I just did this to make it run more quickly. Do you think it's worth parametrizing it and testing both with and without retries?

Reply:
Hmm it seems like testing with retries should cover all the cases without retries too, but testing both also seems fine.

Reply:
Is there a way we can update the config to make the retries faster?

Reply:
Let's just test the retry case with a short timeout. Will update.